lib/content-services/src/lib/category/categories-management/categories-management.component.html

```diff
@@ -21,33 +31,9 @@
             </button>
         </span>
     </div>
-    <div *ngIf="((!categoryNameControlVisible && categories.length)) || categoryNameControlVisible"
-        [hidden]="!categoryNameControlVisible"
-        class="adf-category-name-field">
-        <mat-form-field>
-            <mat-icon matPrefix>search</mat-icon>
-            <mat-label id="adf-category-name-input-label">
```

Comment:
Don't we really need any label for input so user can know what is purpose of that input?